deploy/cluster_role.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I guess `ClusterRole` is required because `consolinks` are kind of goblal resources. Is this true? Is it not enough to add this to the existing `3scale-operator` namespaced role?

Reply:
The consolelinks is clusterwide resource, we must have a ClusterRole.